utilities/common.py

```diff
@@ -238,11 +238,11 @@ def set_subparsers(subparser, name, analysis_label):
         if analysis_label not in axmap:
             raise ValueError(f"Unknown analysis {analysis_label}!")
         subparser.add_argument("--genAxes", type=str, nargs="+", 
-                               default=axmap[analysis_label], choices=["qGen", "ptGen", "absEtaGen", "ptVGen", "absYVGen"],
+                               default=axmap[analysis_label], choices=["qGen", "ptGen", "absEtaGen", "ptVGen", "absYVGen", "helicitySig"],
```

Comment:
Are you sure the helicitySig axes has to go here? It was supposed to be added elsewhere, for technical reasons which right now I don't remember exactly. Could it be why you saw large difference for the theory agnostic fit ?

Reply:
For now I only added it in "choices" in preparation for future updates, I don't think it's causing the problem